### ISSUE TITLE
output heatmap visualisation (5.4) changes everytime I load my network

### ISSUE THREAD

**Issue by JaspervDalen** [fchollet/deep-learning-with-python-notebooks#58]
I tested notebook 5.4 and it works as expected. However when I change the network from VGG16 to my own network (mobilenet with 2 dense and dropout layers behind it), the heatmaps change every time I load the network (load model from keras using .h5 file). Any idea why this happens?

**Comment by JaspervDalen** [comment]
I tried it again with just the mobilenet (applications.mobilenet) trained on imagenet to just replicate the african elephant. The prediction is correct however the heatmap does not work...
